src/index.html

```diff
@@ -7,6 +7,373 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+
+      <a href="#" class="header__logo logo">
+        <img
+            src="images/logo.png"
+            alt="Moyo logo"
+        >
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#Apple" >Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+                class="nav__link"
+                href="#Laptops & Computers"
+                data-qa="nav-hover"
+            >Laptops & Computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Video">Video</a>
+          </li>
+
+        </ul>
+      </nav>
+
+    </header>
+    <main class="catalog">
+      <div class="catalog__container">
+        <div class="catalog__card card" data-qa="card">
+          <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="imac"
+          >
+
+          <h2 class="card__description-name">
+          APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__description-code">Product code: 195434</p>
+
+          <div class="card__reviews-box">
+
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__reviews" >Reviews: 5</p>
+
+          </div>
+
+          <div class="card__price-box">
+            <p class="card__price-text">Price:</p>
+
+            <p class="card__price-number">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >BUY</a>
```

Comment:
```suggestion
          <a
            href="#"
            class="card__button"
            data-qa="card-hover"
          >
            BUY
          </a>
```
+do not use caps here, use `text-transform: uppercase` in css.